docs/examples/workflows/script_annotations_artifacts_raw_old.md

```diff
@@ -0,0 +1,80 @@
+# Script Annotations Artifacts Raw Old
```

Comment:
@dejamiko I wonder if the upstream examples can act as the "old" ones, and then the new ones here can primarily focus on the `Annotated` version? 🤔 cc: @elliotgunton

Reply:
I agree we should make annotations the standard method going forward and avoid overloading the examples section (the old examples don't add value once someone knows about the annotation method - it's only good for the walkthrough section).

Reply:
I added them to make sure the annotations generate the same exact yaml as the old method. I am happy to move all those tests to some sort of unit tests instead of polluting examples